pkg/antctl/raw/traceflow/command.go

```diff
@@ -179,34 +196,37 @@ func newTraceflow(client kubernetes.Interface) (*v1alpha1.Traceflow, error) {
 		return nil, fmt.Errorf("source should be in the format of Namespace/Pod or Pod")
 	}
 
-	var dst v1alpha1.Destination
-	dstIP := net.ParseIP(option.destination)
-	if dstIP != nil {
-		dst.IP = dstIP.String()
-		name = getTFName(fmt.Sprintf("%s-%s-to-%s", src.Namespace, src.Pod, dst.IP))
-	} else {
-		var isPod bool
-		var dest string
-		var err error
-		split = strings.Split(option.destination, "/")
-		if len(split) == 1 {
-			dst.Namespace = "default"
-			dest = split[0]
-		} else if len(split) == 2 && len(split[0]) != 0 && len(split[1]) != 0 {
-			dst.Namespace = split[0]
-			dest = split[1]
+	var dst *v1alpha1.Destination
+	if option.destination != "" {
+		dst = new(v1alpha1.Destination)
+		dstIP := net.ParseIP(option.destination)
+		if dstIP != nil {
+			dst.IP = dstIP.String()
+			name = getTFName(fmt.Sprintf("%s-%s-to-%s", src.Namespace, src.Pod, dst.IP))
 		} else {
-			return nil, fmt.Errorf("destination should be in the format of Namespace/Pod, Pod, Namespace/Service or Service")
-		}
-		if isPod, err = dstIsPod(client, dst.Namespace, dest); err != nil {
-			return nil, fmt.Errorf("failed to check if destination is Pod or Service: %w", err)
-		}
-		if isPod {
-			dst.Pod = dest
-		} else {
-			dst.Service = dest
+			var isPod bool
+			var dest string
+			var err error
+			split = strings.Split(option.destination, "/")
+			if len(split) == 1 {
+				dst.Namespace = "default"
+				dest = split[0]
+			} else if len(split) == 2 && len(split[0]) != 0 && len(split[1]) != 0 {
+				dst.Namespace = split[0]
+				dest = split[1]
+			} else {
+				return nil, fmt.Errorf("destination should be in the format of Namespace/Pod, Pod, Namespace/Service or Service")
+			}
+			if isPod, err = dstIsPod(client, dst.Namespace, dest); err != nil {
```

Comment:
Do you think we can use another parameter for destination service?

Reply:
If pod and service share same parameter, the antctl cannot support trace a service which have same name with pod. I'm OK if we can accept this restriction.

Reply:
Got it.